assets/js/modules/tagmanager/setup.js

```diff
@@ -241,8 +241,8 @@ class TagmanagerSetup extends Component {
 			if ( this._isMounted ) {
 				this.setState( {
 					isLoading: false,
-					error: true,
-					message: err.message
+					errorCode: true,
```

Comment:
Shouldn't this be `err.code`? `errorCode` is not a boolean.

Reply:
@felixarntz updated